modules/imaging_uploader/README.md

```diff
@@ -12,3 +24,77 @@ max_execution_time = 10800
 upload_max_filesize = 1024M
 post_max_size = 1024M
 ```
+
+### Uploaded file
+
+The uploaded file is expected to be of one of the following types: 
+`.tgz, .tar.gz or .zip`. The filename should follow the:
+`PSCID_CandID_VisitLabel_*OptionalSuffix*.zip` naming convention for a `.zip` 
+file. It is also expected that the candidate and visit are already created in 
+the database.
+
+
+## Intended Users
+
+The primary users are MRI technicians or site coordinators to
+upload imaging scans for registered LORIS candidates and timepoints.
+
+## Scope
+
+The imaging uploader has the following built-in capabilities to facilitate 
+timely scan insertion into the LORIS database:
+
+1. Upload imaging scans using the `Upload` tab that can subsequently be inserted 
+into the database automatically using a collection of perl scripts provided in 
+[LORIS-MRI](https://github.com/aces/Loris-MRI). The launching of the insertion
+can also be achieved manually
+2. Browse uploaded scans using the `Browse` tab, which displays information 
+about who uploaded the scan, when it was uploaded, and an `UploadID` and 
+`UploadLocation` that can be used to trigger the LORIS-MRI insertion manually
+3. Track scan insertion status through the `Progress` column and a `Log Viewer` 
+which displays in either `Summary` or `Detailed` mode relevant feedback messages
+about insertion success or failure causes
+4. Uploaded scans can be overwritten if insertion is `Not Started` or `Failed` 
```

Comment:
These seem to all be describing the implementation more than the scope. The first sentence with a second sentence about it also being used for browsing scans and tracking scan insertion status is probably enough.